### ISSUE TITLE
can not run

### ISSUE THREAD

**Issue by Chengmeng94** [ElefHead/numpy-cnn#1]
Hello! When I run cnn.py, I received error as following:
FileNotFoundError: [Errno 2] No such file or directory: './data/data_batch_1'

could you please check it?

**Comment by ElefHead** [comment]
Hi, I'm sorry for not making it clearer on the readme, but you're going to have to download mnist or cifar or something and put it in a data folder. I will soon add more instructions, thanks for reaching out.

**Comment by Chengmeng94** [comment]
> Hi, I'm sorry for not making it clearer on the readme, but you're going to have to download mnist or cifar or something and put it in a data folder. I will soon add more instructions, thanks for reaching out.

I put the data, but get another error:
[Errno 2] No such file or directory: 'models/cnn5/conv_1.pickle'

So could you please upload your whole document?

**Comment by ElefHead** [comment]
I think you tried to run it by uncommenting line 39 on in cnn.py. 
That line is to be used only if you have already trained the model once and saved the weights. 

I wrote this code 2 years ago and like I have mentioned on readme, I don't use it practically because, well, tensorflow and pytorch make all of this easy. Because of this, I don't have my models saved. 

Meanwhile, to run the code, just comment line 39 in cnn.py. I just tried it again on a different pc, and it still works. 

Best,
Ganesh
